src/box/lua/config/instance_config.lua

```diff
@@ -455,7 +456,45 @@ local function feedback_validate(data, w)
     w.error('Tarantool is built without feedback reports sending support')
 end
 
-return schema.new('instance_config', schema.record({
+-- Set the annotations for the given schema object based on the field
+-- path accumulated in ctx.path.
+local function set_annotation(schema_obj, ctx)
+    local field_path = table.concat(ctx.path, '.')
+    if annotations[field_path] ~= nil then
+        schema_obj.description = annotations[field_path]
+    end
+end
```

Comment:
Since we want to cover all the options with some (at least brief) description, I would replace the if operator with the assertion. It allows to spot a missed description when adding a new option into the instance config. Please, provide a good diagnostic for this assertion fail: which option needs a description, so a developer easily find what he/she need to do.

We anyway need a test for missed descriptions, because there are cluster options that are out of the instance config. But this assertion that works on tarantool loading would be useful to spot the problem earlier during development.

Reply:
Replaced with assertion. Test added.